src/create-model.jl

```diff
@@ -217,6 +220,17 @@ function create_model(graph, representative_periods, dataframes; write_lp_file =
     Fi = filter_flows(:investable, true)
     Ft = filter_flows(:is_transport, true)
 
+    # Create subsets of storage assets by storage type
+    As_short = intersect(As, filter_assets(:storage_type, "short"))
+    As_long  = intersect(As, filter_assets(:storage_type, "long"))
+
+    # Check if all assets in As are in As_short and As_long, if not assign the missing ones to short by default
+    all_storage_assets_in_short_and_long = all(a -> a ∈ union(As_short, As_long), As)
```

Comment:
This is very small here, but it might become more common in the future. Instead of `a in union(A, B)`, use `a in A || a in B`, otherwise the memory for `union(A, B)` is created.

Reply:
It is no longer in the code, but it is good to keep it in mind.